### PR TITLE
Implement lifecycle node

### PR DESCRIPTION
**Pull request by guilyx**
The Scanner node is wrapped around in a new node that inherits from the lifecycle class. We now are able to use transitions and states for the sick safetyscanner.

### CODE REVIEW

**Comment by SomaGallai** [comment]
The Lifecycles has been tested and ready to merge.

**Comment by guilyx** [comment]
👍🏾

File: launch/sick_safetyscanners2_launch.py [hunk 1]
```diff
@@ -5,8 +5,8 @@ def generate_launch_description():
     return LaunchDescription([
         Node(
             package="sick_safetyscanners2",
-            executable="sick_safetyscanners2_node",
-            name="sick_safetyscanners2_node",
+            executable="sick_safetyscanners2_lifecycle_node",
```
Comment (guilyx) [inline launch/sick_safetyscanners2_launch.py]: this should be the normal node

File: launch/sick_safetyscanners2_launch.py [hunk 2]
```diff
@@ -5,8 +5,8 @@ def generate_launch_description():
...
+            name="sick_safetyscanners2_lifecycle_node",
```
Comment (guilyx) [inline launch/sick_safetyscanners2_launch.py]: same as above

**Review by guilyx — COMMENTED** [review]
@SomaGallai

File: launch/sick_safetyscanners2_launch.py (hunk 2 above)
Comment (SomaGallai) [reply launch/sick_safetyscanners2_launch.py]: fixed

**Comment by guilyx** [comment]
👍🏾

**Pull request merged**